src/ethereum_clis/transition_tool.py

```diff
@@ -235,6 +250,8 @@ def _evaluate_filesystem(
             input_paths["env"],
             "--input.txs",
             input_paths["txs"],
+            "--input.config",
+            input_paths["config"],
```

Comment:
This will break client tools that do not support this parameter yet.

The way we handle this is by implementing a `ClassVar` in `TransitionTool` that could default to `False` (example here: https://github.com/ethereum/execution-spec-tests/blob/56ea9f2fdb157630101ecced198afae87752eae0/src/ethereum_clis/transition_tool.py#L75), and then override it to `True` only for evmone.